themes/src/main/resources-community/theme/base/login/messages/messages_de.properties

```diff
@@ -369,6 +369,10 @@ webauthn-error-user-not-found=Unbekannter Benutzer, der mit dem Security-Token a
 # Identity provider
 identity-provider-redirector=Mit einem anderen Indentit\u00E4tsprovider verbinden
 identity-provider-login-label=Oder anmelden mit
+idp-email-verification-display-name=E-Mail Verifizierung
+idp-email-verification-help-text=Best\u00E4tigen Sie den Besitz des Benutzerkontos \u00FCber den Empfang einer E-Mail
```

Comment:
as these are german translations, i'll continue my review in german:

Hi @aruehl,  Ich würde vorschlagen, hier bei `idp-email-verification-help-text` etwas kürzeren Text zu nehmen. Vorschlag: `Best\u00E4tigen Sie Ihr Konto \u00FCber den Empfang einer E-Mail` 

Ich bin nicht gnaz im Kontext drin, aber im Ticket #12575 scheint da recht wenig Platz zu sein in dem Feld für den Hilfetext, und ich persönlich finde dass der Text genauso klar ist.

Reply:
Der Platz reicht aus
![grafik](https://user-images.githubusercontent.com/3062585/174628472-0eb8c7ae-2aa8-4d94-9364-107fadd1ada0.png)

ich kann es aber auch nochmal anpassen, wenn du das möchtest

Reply:
ok, works for me ;) danke für das Beispiel!

Reply:
habe es angepasst
![grafik](https://user-images.githubusercontent.com/3062585/174630330-9395827b-94d8-40ec-8b62-d0ff1a61a488.png)

Reply:
Noch besser, damit hat sich auch das CI triggern erledigt (siehe kommentar unten). Thanks :)